ts/components/registration/stages/CreateAccount.tsx

```diff
@@ -77,6 +82,7 @@ export const CreateAccount = () => {
       const newHexPubKey = StringUtils.decode(keyPair.pubKey, 'hex');
 
       dispatch(setRecoveryPassword(mnemonic));
+      dispatch(setPrivateKeyBytes(keyPair.ed25519KeyPair.privateKey));
```

Comment:
idk if that's uint8array or not, but redux doesn't like it if yes. Can you store it as hex instead (sodium exposes from_hex/to_hex)